src/main/client/apply.c

```diff
@@ -171,6 +171,7 @@ PyObject *AerospikeClient_Apply_Invoke(AerospikeClient *self, PyObject *py_key,
         PyObject *py_err = NULL;
         error_to_pyobject(&err, &py_err);
         PyObject *exception_type = raise_exception_old(&err);
+        set_aerospike_exc_attrs_using_tuple_of_attrs(exception_type, py_err);
```

Comment:
Suggestions (non blocking): This block seems to be repeated in many places, maybe it could be a function?

Reply:
Hey Dylan, all those blocks of code are using `raise_exception_old` and I didn't have time to refactor them into `raise_exception`, yet. I will address this in the future, but it's a lot of work because these blocks of code are present in a lot of places